site/_apps/hn-mithril.txt

```diff
@@ -0,0 +1,26 @@
+---
+weight: 12
```

Comment:
Recent PR implementation had a weight of 15, could you  +1 one that here so it continues the sequence?